helm/wrongsecrets-ctf-party/values.yaml

```diff
@@ -209,6 +209,8 @@ wrongsecretsCleanup:
   enabled: true
   # -- Specifies when Juice Shop instances will be deleted when unused for that period.
   gracePeriod: 2d
+  # -- Specifies if the clean up job should delete the outdated namespaces or just report them. Set to false to only report outdated namespaces.
+  isDelete: false
   # -- Cron in which the clean up job is run. Defaults to once in an hour. Change this if your grace period if shorter than 1 hour
```

Comment:
```suggestion
  # -- Cron in which the clean up job is run. Defaults to once in a quarter. Change this if your grace period if shorter than 1 hour. See https://crontab.guru/#0,15,30,45_*_*_*_* for more details.
```

Reply:
`  # -- Cron in which the clean up job is run. Defaults to once in a quarter. Change this if your grace period if shorter than 15 minutes. See https://crontab.guru/#0,15,30,45_*_*_*_* for more details.`

Reply:
Done